samples/kubernetes/end2end/docs/dashboard.md

```diff
@@ -0,0 +1,22 @@
+## What's in the Weblogic Dashboard
+### Overview
+The WebLogic Dashboard helps you know what's going on inside WebLogic servers, how much traffic they're getting, how they're performing, any errors etc.
```

Comment:
Replace this sentence with:
The WebLogic Server Grafana dashboard provide a visualization of WebLogic runtime metrics allowing you to monitor and diagnose runtime deployment of WebLogic.